InsiteTeamTask_A/Startup.cs

```diff
@@ -25,24 +26,35 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services.AddControllers();
+
+            // same-origin policy states that a web broswer will only communicate between two URL's if the they belong to the same origin. A client app cannot communicate witha server app if they belong to a different origin.
+            services.AddCors(options => options.AddDefaultPolicy(builder => builder.WithOrigins("http://localhost:4200").AllowAnyHeader()));
```

Comment:
Your client value here should be stored in a string array in your `appsettings.json`, so that when I need to change the client port (4200 is the Angular default so it's very likely I already have something there), I don't need to change server code and recompile.